tasks/generate_mapping.py

```diff
@@ -230,7 +230,12 @@ def _write_attached_storage_domains(f, dc_service, dc):
                     % attached_sd.discard_after_delete)
             f.write("  dr_primary_path: %s\n" % attached_sd.storage.path)
             f.write("  dr_primary_address: %s\n" % attached_sd.storage.address)
+            if (attached_sd._storage.type == types.StorageType.POSIXFS):
+                f.write("  dr_primary_vfs_type: %s\n"
+                        % attached_sd.storage.vfs_type)
             _add_secondary_mount(f)
+            if (attached_sd._storage.type == types.StorageType.POSIXFS):
+                f.write("  dr_secondary_vfs_type: \n")
```

Comment:
Should be here:
```python
f.write("  dr_primary_vfs_type: %s\n"
                        % attached_sd.storage.vfs_type)
```

Reply:
we acctually want the secondary type here since _add_secondary_mount(f) in line 236 adds the scondary mount options.
I thought to add this as part of the _add_secondary_mount method but I wanted to avoid sending the storage type as the parameter.
I can add it to the method if you think it will be more readable

Reply:
I will change it to be part of the _add_secondary_mount method